docs/resources.rst

```diff
@@ -36,6 +36,9 @@ Solidity Integrations
     * `Superblocks Lab <https://lab.superblocks.com/>`_
         Browser-based IDE. Built-in browser-based VM and Metamask integration (one click deployment to Testnet/Mainnet).
 
+    * `Solidity IDE <https://github.com/System-Glitch/Solidity-IDE>`_
+        Browser-based IDE with integrated compiler, integrated Ganache and local file system support.
```

Comment:
Is it really browser or electron based?

Reply:
It is a vue js interface interacting with a nodejs server (which handles the file system and the compilation)

Reply:
Small suggestion for tiny readability improvement

```suggestion
        Browser-based IDE with integrated compiler, Ganache and local file system support.
```

Reply:
So do I open a URL in a browser of my choice or how does it work?

Reply:
Yes. When running `npm run ide`, the solc server and ganache are started and the IDE opens in your default browser.